pylint_django/tests/test_app/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class TestAppConfig(AppConfig):
+    name = 'test_app'
```

Comment:
I believe all of this should be under `input/`. I can see a `models/author.py` module in there.

Reply:
Ah okay, I wasn't sure if this belonged in there since it's moreso setup than something to test. I'll move this into `input/`.